File: python/shipment_item_fulfillment.py
import sdk.python.entities
from sdk.python.entities import Property


class ShipmentItemFulfillment(sdk.python.entities.Entity):

    resource = '/shipment_item_fulfillments/'
    json_name = 'shipmentItemFulfillment'

    assignment = Property('sdk.python.assignments.Assignment')
    job = Property('sdk.python.jobs.Job')

File: javascript/shipment_item_fulfillment.js
import { generateUUID } from './uuid.js';
import { addPropertyTo } from './model.js';
import { Assignment } from './assignment.js';
import { Job } from './job.js';

export function ShipmentItemFulfillment() {
    this.resource = '/shipment_item_fulfillments';
    this.json = 'shipmentItemFulfillment';
    this.temporaryId = generateUUID();

    addPropertyTo(this, 'assignment', Assignment);
    addPropertyTo(this, 'job', Job);
}
